compose.yml

```diff
@@ -12,37 +12,33 @@ services:
     environment:
       - DRACOR_API_BASE=http://localhost:8088/api/v1
     ports:
-      - "8080:8080"
+      - 8080:8080
     volumes:
       - existdata:/opt/exist/data
       - .:/tmp/export
     depends_on:
       - fuseki
       - metrics
   fuseki:
-    image: "dracor/dracor-fuseki"
-    build: https://github.com/dracor-org/dracor-fuseki.git#v1.1.0
```

Comment:
For development purposes I would still like to be able to easily build the individual services from their sources. Should we add the build properties to compose.dev.yml now (probably pointing to #main) or could we leave them in here?

Reply:
That is your choice.

I just wanted to propose this, because I had many iterations testing the API in the CI of my fork. It took a long time to build the services every time.

Reply:
I moved the build information to compose.dev.yml.